content/help/ops/setup/injection/index.md

```diff
@@ -0,0 +1,242 @@
+{{< /text >}}
+
+"x509: certificate signed by unknown authority" errors are typically
+caused by an empty `caBundle` in the webhook configuration.
+
+1. Verify the `caBundle` in the mutatingwebhookconfiguration matches
+   the root certificate mounted in the `istio-sidecar-injector` pod.
+
+{{< text bash >}}
+$ kubectl get mutatingwebhookconfiguration istio-sidecar-injector -o yaml -o jsonpath='{.webhooks[0].clientConfig.caBundle}' | md5sum
+4b95d2ba22ce8971c7c92084da31faf0  -
+$ kubectl -n istio-system get secret istio.istio-sidecar-injector-service-account -o jsonpath='{.data.root-cert\.pem}' | md5sum
+4b95d2ba22ce8971c7c92084da31faf0  -
+{{< /text >}}
+
+The CA certificate should match. If they do not, restart the
+sidecar-injector pods.
+
+{{< text bash >}}
+$ kubectl -n istio-system patch deployment istio-sidecar-injector \
+    -p "{\"spec\":{\"template\":{\"metadata\":{\"labels\":{\"date\":\"`date +'%s'`\"}}}}}"
+deployment.extensions "istio-sidecar-injector" patched
+{{< /text >}}
+
+In theory, the `caBundle` could be temporarily out of date with what
+the injector is used if multiple webhook replicas are in use. This
+should be a transient error state if the CA cert is rotated or install
+is re-installed.
+
+### `no such hosts` or `no endpoints available` errors in deployment status
+
+Injection is fail close. If the `istio-sidecar-injector` pod is not ready, pods
+cannot be created. In such cases you’ll see an error about `no such
+host` (Kubernetes 1.9) or `no endpoints available` (>=1.10).
+
+Kubernetes 1.9:
+
+{{< text text >}}
+Internal error occurred: failed calling admission webhook "istio-sidecar-injector.istio.io": \
+    Post https://istio-sidecar-injector.istio-system.svc:443/admitPilot: dial tcp: lookup \
+    istio-sidecar-injector.istio-system.svc on 169.254.169.254:53: no such host
+{{< /text >}}
+
+Kubernetes 1.10:
+
+{{< text text >}}
+Internal error occurred: failed calling admission webhook "istio-sidecar-injector.istio.io": \
+    Post https://istio-sidecar-injector.istio-system.svc:443/admitPilot?timeout=30s: \
+    no endpoints available for service "istio-sidecar-injector"
+{{< /text >}}
+
+{{< text bash >}}
+$  kubectl -n istio-system get pod -listio=sidecar-injector
+NAME                            READY     STATUS    RESTARTS   AGE
+istio-sidecar-injector-5dbbbdb746-d676g   1/1       Running   0          2d
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl -n istio-system get endpoints istio-sidecar-injector
+NAME           ENDPOINTS                          AGE
+istio-sidecar-injector   10.48.6.108:9093,10.48.6.108:443   3d
+{{< /text >}}
+
+If the pods or endpoints aren't ready, check the pod logs and status
+for any indication about why the webhook pod is failing to start and
+serve traffic.
+
+{{< text bash >}}
+$ for pod in $(kubectl -n istio-system get pod -listio=sidecar-injector -o jsonpath='{.items[*].metadata.name}'); do \
+    kubectl -n istio-system logs ${pod} \
+done
+{{< /text >}}
```

Comment:
You should be able to combine both of these text block together, the rendering should look OK.